src/Validator/ValidatorInterface.php

```diff
@@ -21,7 +21,7 @@
      *
      * @return bool
      */
-    public function isValid();
+    public function __invoke();
```

Comment:
Previous format was correct. Implementators was using `[new $validator($data), 'isValid']` which is callable too